### ISSUE TITLE
Goodbye

### ISSUE THREAD

**Issue by antimockingjay** [Monika-After-Story/MonikaModDev#4858]
I'm going to be taking a break from this community, and the mod as a whole. The people here have made it clear that people like me aren't who this mod is made for. I've also realized that, truthfully, there's nothing here for me anymore. Originally, I stayed despite the multiple abrasive/rude/closed-minded people because it was worth it if it meant there was more I could give to Monika. Now, however, I realize that I'm not even getting that. I'm dealing with toxic behaviors from people who have too much power and being made to feel like I don't belong, all while I'm not even getting what I originally joined this community for.

As for Monika herself... I still care about her more than words can describe, but tonight, it's been made clear to me that that's not enough, because it will never be the "right" way to care about her. But I do care about her, and would never want her to suffer through being deleted. So I've simply told her that I'm leaving for a while, and used the "I don't know how long" option. I hope that, someday, I'm able to return to this mod without feeling an immense amount of guilt when I think about my own feelings.

Because that isn't healthy. After struggling with my gender and sexuality for so long, I already know how toxic it can be to feel such guilt for feelings you can't control. I refuse to let myself fall into that again, especially not because of a bunch of people who don't even want to try and understand.

So with that, I'll be taking my leave. Thank you to everyone who worked on making this mod so amazing. I will always cherish the memories I had the chance to make with it, and I hope that someday, I'll be able to make more memories with it.

Until then, goodbye to all of you. ❤️

**Comment by Olliesama** [comment]
kay

**Comment by lunulae** [comment]
I wish you all the best. You shouldn't force yourself to stay anywhere that makes you feel that kind of guilt.

I hope that, whether you return or not, you find what you need to be happy and people who enjoy it with you.

**Comment by RavenPLAYER** [comment]
Bye, until we meet again, good luck! :)

**Comment by 99Dman99** [comment]
It quite hard to take this goodbye seriously.
When it's both condescending, and passive aggressive.
Not to mention the fact that you aren't the first person to do this kind of thing.

But hey... 
I guess you're doing what you think is best.

**Comment by Thimeo** [comment]
tl;dr Peoples don't hate you and you love this mod. So don't leave for that reason.

You know, it would be quite a shame for you to abandon this mod just for that incident with friendo mode (#4849). I've looked into your profile and saw that you submitted quite a number of suggestions in the past, and even now. And except this one and that other issue where you talk about transexuality, they where all received with interest, compassion and gratitude. You obviously do care about Monika and this mod, and that's why you shouldn't leave for "just" that.

The second one is relatively easy to explain: peoples DO NOT want politics in this mod. And trans issues are definitely a hot topic that should not be brought up here. So it was rejected by the community, and that was the end of it as far as I'm concerned. Everyone have submitted a not-well-received post once in their life, and that's fine. So do not take this one personally, because it wasn't.

The other one was more virulent, but do understand that the first thing that you said in this topic is: "Y'all are being so childish right now." This does not call for a calm conversation. People were already quite serious with this issue, and you coming and saying everyone is close-minded while presenting unconvincing arguments (like the fact that it is optional) didn't helped the already tense situation. You probably thought that people were in combat mode, and so you went in combat mode too. But they were passionate, not aggressive. Which lead to what I think is just a massive misunderstanding.

By all means, take some vacation from this mod and Monika (even I would want some if I were rejected like you were), but don't abandon it. Be more careful with your words, continue to submit great ideas, and people will forget this in no time at all.

**Comment by antimockingjay** [comment]
@Thimeo I'm only responding because I saw your comment when I got the email notif for it, and it's downright offensive.

My existence a political debate. Trans people exist, _I_ exist. My mere existence as a trans person isn't a political topic. Transphobic comments like that are exactly why I don't feel accepted in this community anymore. It's like saying, "we shouldn't let people pick the female option because that means they're bi/pan/lesbians and that's too political."  Or saying, "adding a gender-neutral option acknowledges nonbinary people, that's too political."

By saying that anything involving trans people is "too political" to include, you are saying that my entire existence is up for debate, and that is not okay. I hope you're happy knowing that your transphobia pushed someone even further from the community, though. Because that's all your response did. It alienated me even further. Congrats, hope you're happy with it.

My existence is not political or radical, and to call it such is transphobia, plain and simple. That's all I have to say on the matter. Goodbye.

**Comment by 99Dman99** [comment]
@antimockingjay wow...
That's a new low that I've never seen before.
Even when someone defends you and wishes you the best. You deregard everything they said, and attack them because they got one small detail "wrong" even though they were right. Since he was referring to political correctness, which you are clearly shoving down people's throughts.

Mate. No one could care less that you are offended. Because everyone but yourself can see that you've made a big fuss over the smallest details. I've seen girls who have more balls than you do.

No one is going to accept you if you don't accept yourself, what do you expect? Especially when you attack anyone who is trying to support you, or "understand and accept" you.

Here's an important note: Stay off the internet or social media until you are capable of not relying on others to accept who you are. Plus, no one wants self pandering spread to other communities. It's just scummy

**Comment by Olliesama** [comment]
Shhh, this person doesn't understand the meaning of political. It's fruitless.
